### PR TITLE
Budget-App

### PR DESCRIPTION
**Pull request by cvalencia1991**
# In this milestone, I did the following activities 
- Use ruby gems as a software package system.
- Install Ruby on Rails framework.
- Use controllers to handle requests and render empty views.
- Use layouts and templates for shared content.
- Secure app from n+1 problems.
- Build a web app that requires the user to log in.
- Use devise gem for authentication.
- Limit access to web app resources based on authorization rules.
- Deploy Project using Render
- Analyze in writing why you have made a coding choice using one structure over another.
- Testing the models and request for each component 

image

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```
Comment (1GORDON) [inline app/views/layouts/application.html.erb:1]: Great job @cvalencia1991 🤝

- [ ] Great job on implementing the functionality. However, I would suggest you stick to the design guidelines as much as possible.
- Given design
given-design
- Your design
your-design

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @cvalencia1991 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] Great job presenting the project features in your video presentation. However, as suggested in the project requirements, you forgot to showcase the piece of code you are proud of writing. Kindly highlight a piece of the code you find interesting or proud of making.

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (cvalencia1991) [reply app/views/layouts/application.html.erb:1]: but the requirements is working well ??

**Review by brytebee — APPROVED** [review]
Hello, @cvalencia1991👋 ,

------

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

------

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by cvalencia1991** [comment]
thank you 👍

**Pull request merged**